include/ts/apidefs.h.in

```diff
@@ -771,6 +771,7 @@ typedef enum {
   TS_CONFIG_HTTP_CACHE_CACHE_RESPONSES_TO_COOKIES,
   TS_CONFIG_HTTP_CACHE_IGNORE_AUTHENTICATION,
   TS_CONFIG_HTTP_CACHE_CACHE_URLS_THAT_LOOK_DYNAMIC,
+  TS_CONFIG_HTTP_CACHE_POST_METHOD,
```

Comment:
Unless this is not going in a 9.x release, I think you need to add new enum values here at the end, to avoid the need to rebuild plugins built for ATS9, no?